control-plane/csi-driver/src/bin/node/dev/nvmf.rs

```diff
@@ -307,6 +309,16 @@ pub(crate) fn check_nvme_tcp_module() -> Result<(), std::io::Error> {
     Ok(())
 }
 
+/// Check for the presence of nvme tcp kernel module.
+/// TODO: Handle the case where this(and for that matter nvme_tcp too)
```

Comment:
we need to fix the init-containers as well, which only check for module and not builtin

Reply:
yeah, will do that as separate task. We have ticket for that AFAIK.